src/pages/office/cat-cpc/components/combination/combination.ts

```diff
@@ -0,0 +1,35 @@
+import { Component, Input } from '@angular/core';
+
+import {
+  Combination,
+} from '../../../../../shared/constants/cpc-questions/cpc-question-combinations.constants';
+
+@Component({
+  selector: 'combination',
+  templateUrl: 'combination.html',
+})
+export class CombinationComponent {
+
+  // combinationAdditionalText: any;
+
+  @Input()
+  combination: Combination;
```

Comment:
not sure that is the correct interface, the interface for 'LGV1' for example is 'CombinationCodes'

Reply:
changed to match input